ant-peers-acquisition/Cargo.toml

```diff
@@ -10,6 +10,7 @@ repository = "https://github.com/maidsafe/autonomi"
 version = "0.5.7"
 
 [features]
+default = ["network-contacts"]
```

Comment:
Could we just lose this as a 'feature'? I think that was very excessive anyway.

Reply:
Yes! We'd remove this.
The `network-contacts` just indicates if we want to fetch the `mainnet` contacts or not.

This can be replaced by a simple `--testnet` flag.